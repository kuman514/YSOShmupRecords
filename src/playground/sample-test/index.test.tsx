import { cleanup, render } from '@testing-library/react';
import 'jest-styled-components';
import styled from 'styled-components';
import { beforeEach, describe, expect, it } from 'vitest';

const SampleButton = styled.button`
  background-color: red;
`;

describe('Sample test', () => {
  beforeEach(() => {
    cleanup();
  });

  it('should have a snapshot match', () => {
    const { container } = render(<SampleButton />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
