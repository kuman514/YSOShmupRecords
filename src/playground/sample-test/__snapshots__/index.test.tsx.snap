// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Sample test > should have a snapshot match 1`] = `
.c0 {
  background-color: red;
}

<button
  class="c0"
/>
`;
